### PR TITLE
Approve before notifying requestor

### PR DESCRIPTION
**Pull request by nieznanysprawiciel**

### CODE REVIEW

File: core/market/src/db/dao/agreement.rs [hunk 1]
```diff
@@ -209,6 +209,33 @@ impl<'c> AgreementDao<'c> {
...
 
+    pub async fn create_approve_event(&self, id: &AgreementId) -> Result<(), AgreementDaoError> {
+        let id = id.clone();
+
+        do_with_transaction(self.pool, move |conn| {
+            let agreement: Agreement =
+                market_agreement.filter(agreement::id.eq(&id)).first(conn)?;
+
+            // Always Provider approves.
+            create_event(conn, &agreement, None, Owner::Provider)?;
```
Comment (tworec) [inline core/market/src/db/dao/agreement.rs:220]: remove event from approve

**Review by tworec — APPROVED** [review]
lgtm

**Comment by nieznanysprawiciel** [comment]
This is second solution that we reject.
Instead: https://github.com/golemfactory/yagna/pull/1015

**Comment by tworec** [comment]
Yes, to comply with Market Specification we need to fix it other way. Closed in favor of #1015

**Pull request closed without merging**